src/banner/BannerCheckpoint.jsx

```diff
@@ -0,0 +1,125 @@
+import React from 'react';
```

Comment:
what do you think about renaming this `BannerProvider`

Reply:
I'm fine with that. I'll update the name throughout

Reply:
renamed: https://github.com/cerner/terra-application/pull/45/commits/58171140c932a65d683734578709bac27efde528